test/support/utilities.rb

```diff
@@ -5,13 +5,11 @@ def configure!
   end
 
   after do
-    Restforce::DB::FieldProcessor.reset
-    Restforce::DB::Registry.clean!
-    Restforce::DB.last_run = nil
-    Restforce::DB.logger = nil
```

Comment:
Yes; the logger's on the configuration, which gets wiped as part of the `.reset!` invocation.